contributors.yml

```diff
@@ -64,3 +64,5 @@
 - underager
 - vijaypushkin
 - vikingviolinist
+- KutnerUri
```

Comment:
Let's remove `KutnerUri` which is already in here above - likely a merge conflict issue